### ISSUE TITLE
Is it possible to fine-tune Geolayout LM only for entity labelling ?

### ISSUE THREAD

**Issue by SR1608** [AlibabaResearch/AdvancedLiterateMachinery#55]
The data on which we want to finetune Geolayout LM model doesnt have key-value pairs.
For ex Name,Address,Dates etc 

Is it possible to ignore entity linking part and just train the model for entity labelling task?

**Comment by ccx1997** [comment]
Yes, you may set the linking loss to 0 mannually, and ignore the linking part.
